resources/icons/or.svg

```diff
@@ -0,0 +1,18 @@
+<svg width="28" height="29" viewBox="0 0 28 29" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
so & is OR ?

Reply:
So I've asked about it Anatoly, about maybe it will be better to make additional icon for or, he said that's leave as it is right now, grey icon for 'or' and blue for 'and'